Sources/NIOSSL/SSLErrors.swift

```diff
@@ -157,6 +157,12 @@ internal extension BoringSSLError {
     }
 }
 
+public enum NIOSSLSecureBytesError: Error {
```

Comment:
We only use this in the tests: can we just define it there instead of adding this as new API?

Reply:
Forgot to add this as part of 8007b79, coming right up.

Reply:
Done